### ISSUE TITLE
Filter list on Tags and select Group Monitors by Tags

### ISSUE THREAD

**Issue by Bodge-IT** [louislam/uptime-kuma#1609]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

A method to massively improve the organisational capability of the interface

### ✔️ Solution

In status page, selecting a tag from Monitor select list, should pull in all monitors with that tag, dynamically.

### ❓ Alternatives

status quo...eurgh.

**Comment by EDIflyer** [comment]
Ah - I thought I was doing something wrong at first as I'd set tags but couldn't find a way to set them on the status page 😀 
Totally agree, this would be a fantastic addition - if you were able to add desired tags when configuring a status page then this would be an incredibly quick way to set up pages for different tags/combinations of tags 🥳

**Comment by smartyparts** [comment]
Yes, Please. Please let us build status pages by Tag. That way as we add/remove checks the Status Page will automatically be current.

**Comment by jmerhar** [comment]
I see many duplicate requests for this same feature. For anyone with a lot of monitors, like myself, this is quite essential. It would make my life a lot easier if I could just add a new monitor, tag it, and it would automatically appear on the relevant status pages. Keeping status pages up-to-date manually is very tedious for a large number of monitors.
